bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableClusterUtilities.java

```diff
@@ -217,7 +328,7 @@ public synchronized int getClusterNodeCount(String clusterId, String zoneId) {
    * @return the {@link Cluster} if it was set. If the cluster is not found, throw a {@link
    *     NullPointerException}.
    */
-  public synchronized Cluster getCluster(String clusterId, String zoneId) {
+  public Cluster getCluster(String clusterId, String zoneId) {
     Cluster response = null;
     for (Cluster cluster : getClusters().getClustersList()) {
```

Comment:
I think my point is that with v2 client, getClusters() could return all clusters within a project (belong to different instances).

The following is valid and "cluster1" cluster ID will match both of the clusters.
Instance1:cluster1
instance2:cluster1